src/event_pusher/mod_event_pusher_push_plugin_defaults.erl

```diff
@@ -58,15 +58,23 @@ sender_id(From, Packet) ->
                            Services :: [mod_event_pusher_push:publish_service()]) ->
     mongooseim_acc:t().
 publish_notification(Acc0, From, #jid{lserver = Host} = To, Packet, Services) ->
+    MessageCount = try mongoose_acc:get(inbox, unread_count, Acc0) of
```

Comment:
`MessageCount` suggests the value is a number, but it's stored as a binary - we should keep it as a number and only convert to/from a binary on system boundaries.